proteus/MeshAdaptPUMI/SizeField.cpp

```diff
@@ -59,8 +59,9 @@ int MeshAdaptPUMIDrvr::calculateSizeField()
     changes in the smoothed size are less than 50% of hmin.
    */
   double err_h_max=hmax;
-  int its=0;
-  while (err_h_max > 0.5*hmin && its < 200)
+  //int its=0;
+  //while (err_h_max > 0.5*hmin && its < 200)
+  for(int its=0;its<200;its++)
     {
```

Comment:
Yes, I was running into issues where a collective call was not completing. My understanding is that some ranks seemed to be stuck in the while loop but other ranks have left, which is an issue since the smoothing process is a collective operation built on the following abstract operator: 

https://scorec.rpi.edu/~dibanez/core/classapf_1_1CavityOp.html#a5d04176d38cd0e0a54e9c6e9a1a6f45a

I'm coding up a mesh gradation function currently, so this section  of code will likely be unnecessary soon regardless.